### ISSUE TITLE
Show Password “eye icon” not reachable by screen reader or keyboard

### ISSUE THREAD

**Issue by Sillymak** [pwm-project/pwm#651]
**Describe the bug**
The "eye icon" to show password is not reachable by screen readers or keyboard

Is it possible to change this to a button or something reachable by screen reader?

**Comment by jrivard** [comment]
Not sure why this would matter to a screen reader?   it merely allows the password field to be visible, which it seems like a screen reader would offer this as basic functionality of its own?

Is this toggle enhibiting this functionality in the reader?

**Comment by Sillymak** [comment]
We can omit the screen reader portion. We are just concerned with keyboard users reaching the show password button.
